site/src/components/InfoTooltip/InfoTooltip.tsx

```diff
@@ -26,8 +26,14 @@ export const InfoTooltip: FC<InfoTooltipProps> = ({
 
 	return (
 		<HelpTooltip>
-			<HelpTooltipIconTrigger size="small" css={styles.button}>
-				<HelpTooltipIcon css={{ color: iconColor }} />
+			<HelpTooltipIconTrigger
+				size="small"
+				className="opacity-100 hover:opacity-100"
+			>
+				<HelpTooltipIcon
+					style={{ "--icon-color": iconColor }}
+					className="text-[var(--icon-color)]"
```

Comment:
```suggestion
					style={{ color: iconColor }}
```

this seems like an over optimization to me